### DISCUSSION TITLE
Documentation of MPD sticker usage

### DISCUSSION THREAD

**Discussion by jcorporation** [MusicPlayerDaemon/MPD#1887] [General]
I created a repository to document the usage of MPD stickers across various MPD clients. 

The ultimate goal of this project is to create a standard for common sticker names.

All mpd client developers and other mpd users are very welcome to join this project!

https://github.com/jcorporation/mpd-stickers

The project is at the very beginning, so at the moment only myMPD is listed there. I am the developer of myMPD.

**Comment by MaxKellermann** [comment]
What do you think of moving the document into the MPD repository? As an appendix to the MPD documentation, documenting common usage types in a canonical location.

**Reply by jcorporation** [reply]
It would be great, if this becomes a piece of the official mpd documentation. Should we wait, if there was more feedback on this attempt before we are investing more work?

How the workflow would look like? Should we maintain the document in the actual repository and merge it regularly into the mpd repository? I do not want to burden you with extra work.
